util/chplenv/chpl_llvm.py

```diff
@@ -117,6 +117,11 @@ def check_llvm_packages(llvm_config):
     usr_include_clang_ok = False
 
     include_dir = run_command([llvm_config, '--includedir']).strip()
+    if "/nix/store" in include_dir:
+        # Dependencies are managed by Nix, and llvm, clang, etc. are all
+        # different packages. Hence, we cannot rely on clang/Basic/Version.h
+        # residing in the include_dir returned by llvm-config.
```

Comment:
Could you update the PR message to indicate that this PR disables some checking for the existence of a clang developer package when nix is detected?

Reply:
Added to the description.